UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterPlayer.cs

```diff
@@ -118,4 +136,79 @@ public void GetUp()
 			}
 		}
 	}
+
+	//Handled via UpdateManager
+	void UpdateMe()
+	{
+		//Server Only:
+		if (CustomNetworkManager.Instance._isServer)
+		{
+			tick += Time.deltaTime;
+			if (tick > tickRate)
+			{
+				tick = 0f;
+				CalculateStun();
+			}
+		}
+	}
+
+	public void CheckTileSlip()
+	{
+		if (metaDataLayer.IsSlipperyAt(transform.position.CutToInt()))
```

Comment:
```suggestion
		if (metaDataLayer.IsSlipperyAt(Position))
```

Reply:
![HhlrpIk](https://user-images.githubusercontent.com/28093344/55063587-26c5c300-50dd-11e9-9d82-facd3f0b62dc.png)
Position is off by 1 on the X and Y axis, so this change makes the top right tile of the tile cleaned slippery instead.
Since Position is read in 21 different places, this is seems pretty strange to me.